pkg/cmd/roachtest/tests/java_helpers.go

```diff
@@ -192,7 +193,9 @@ func parseAndSummarizeJavaORMTestsResults(
 	for i, file := range files {
 		t.L().Printf("Parsing %d of %d: %s\n", i+1, len(files), file)
 		// NB: It is necessary to single quote the file name to prevent
-		// unintentional variable interpolation if the name contains $'s.
+		// unintentional variable interpolation if the name contains $'s. The
```

Comment:
Actually I wonder if we should maybe replace instances of `$` with `\$` instead of single-quoting the file name

Reply:
done